src/index.html

```diff
@@ -22,6 +22,409 @@
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="#apple"
+              class="header__nav-link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#sumsung"
+              class="header__nav-link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#smartphones"
+              class="header__nav-link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+
+          <li
+            class="header__nav-item"
+            data-qa="hover"
+          >
+            <a
+              href="#laptops"
+              data-qa="nav-hover"
+              class="header__nav-link"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#gadgets"
+              class="header__nav-link"
+            >
+              GADGETS
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#tablets"
+              class="header__nav-link"
+            >
+              TABLETS
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#photo"
+              class="header__nav-link"
+            >
+              PHOTO
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#video"
+              class="header__nav-link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="Desctop-img"
+          class="card__img"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__description">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__rating-text">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </article>
+
+      <article
+        class="card"
+        data-qa="card-hover"
```

Comment:
The `data-qa` attribute value 'card-hover' is used here but it should be unique for each card if it's meant for testing purposes. Consider using a more specific value or removing it if not necessary.